src/Symfony/Component/Validator/Resources/bin/sync-iban-formats.php

```diff
@@ -0,0 +1,204 @@
+#!/usr/bin/env php
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+if ('cli' !== \PHP_SAPI) {
+    throw new \Exception('This script must be run from the command line.');
+}
+
+/*
+ * This script syncs IBAN formats from the upstream and updates them into IbanValidator.
+ *
+ * Usage:
+ *   php Resources/bin/sync-iban-formats.php
+ */
+
+error_reporting(\E_ALL);
+
+set_error_handler(static function (int $type, string $msg, string $file, int $line): void {
+    throw new \ErrorException($msg, 0, $type, $file, $line);
+});
+
+echo "Collecting IBAN formats...\n";
+
+$formats = array_merge(
+    (new WikipediaIbanProvider())->getIbanFormats(),
+    (new SwiftRegistryIbanProvider())->getIbanFormats()
+);
+
+printf("Collected %d IBAN formats\n", count($formats));
+
+echo "Updating validator...\n";
+
+updateValidatorFormats(__DIR__.'/../../Constraints/IbanValidator.php', $formats);
+
+echo "Done.\n";
+
+exit(0);
+
+function updateValidatorFormats(string $validatorPath, array $formats): void
+{
+    ksort($formats);
+
+    $formatsContent = "[\n";
+    $formatsContent .= "        // auto-generated\n";
+
+    foreach ($formats as $countryCode => [$format, $country]) {
+        $formatsContent .= "        '{$countryCode}' => '{$format}', // {$country}\n";
+    }
+
+    $formatsContent .= '    ]';
+
+    $validatorContent = file_get_contents($validatorPath);
+
+    $validatorContent = preg_replace(
+        '/FORMATS = \[.*?\];/s',
+        "FORMATS = {$formatsContent};",
+        $validatorContent
+    );
+
+    file_put_contents($validatorPath, $validatorContent);
+}
+
+final class SwiftRegistryIbanProvider
+{
+    /**
+     * @return array<string, array<string, string>>
```

Comment:
this is a wrong type. the values in the returned array are not arrays with string keys and string values. the proper return type is `array<string, array{string, string}>`